content/news/2022-07-29-bevy-0.8/index.md

```diff
@@ -0,0 +1,1131 @@
++++
+title = "Bevy 0.8"
+date = 2022-07-29
+[extra]
+author = "Carter Anderson"
+twitter = "cart_cart"
+github = "cart"
+youtube = "cartdev"
+image = "bevy_terrain.png"
+show_image = true
+image_subtitle = "Bevy-shaped mountains in a Bevy-based Witcher 3 terrain texturing tool built by rmemr"
+image_subtitle_link = "https://codeberg.org/rmemr/w3.terrain-texturing"
++++
+
+Thanks to **X** contributors, **X** pull requests, and our [**generous sponsors**](https://github.com/sponsors/cart), I'm happy to announce the **Bevy 0.8** release on [crates.io](https://crates.io/crates/bevy)!
```

Comment:
I think community reviews should be called out more explicitely, there's a lot of PRs stuck on lack of reviews. This post is pretty much entirely about PRs that have been merged and the authors of these, but people that have helped review and test PRs are not mentioned anywhere. There might be a more appropriate place for that.

```suggestion
Thanks to **X** contributors, community reviewers, **X** pull requests, and our [**generous sponsors**](https://github.com/sponsors/cart), I'm happy to announce the **Bevy 0.8** release on [crates.io](https://crates.io/crates/bevy)!
```

Reply:
Agreed. We should also consider a call to action.

Reply:
Yeah I dig this, although I think it makes sense to put it after the pull requests number to "contextualize" it. Ill commit this as-is and then re-order in my next pass.

Reply:
Something to the effect of:

"The most effective to get your favourite new feature or bugfix into Bevy is reviewing pull requests, so please head on over. We believe in you!"

?

Not completely sure about the phrasing, but it is _the_ most effective way to contribute at this point

Reply:
Something like that could be added too, but my intention here was to thank the existing reviewers. .